config/locales/sufia.en.yml

```diff
@@ -211,8 +213,3 @@ en:
         description: "Usage information for abstract or summary"
         tag: "Usage information for keyword"
         rights: "Usage information for rights"
-
-  hydranorth:
```

Comment:
Sorry I thought the rebase would take care of it. Probably didn't do it
right. It's fixed now.

On Fri, Mar 27, 2015 at 5:01 PM, Peter Binkley <notifications@github.com>
wrote:

> In config/locales/sufia.en.yml
> <https://github.com/ualbertalib/HydraNorth/pull/154#discussion_r27337224>:
>
> > @@ -211,8 +213,3 @@ en:
> >          description: "Usage information for abstract or summary"
> >          tag: "Usage information for keyword"
> >          rights: "Usage information for rights"
> > -
> > -  hydranorth:
>
> And this section is needed by Chris's All Files change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/154/files#r27337224>.
>